api/v1/flow/flow.proto

```diff
@@ -127,7 +127,7 @@ enum FlowType {
 enum AuthType {
     NONE = 0;
     NULL = 1;
-    MTLS_SPIFFE = 2;
+    SPIFFE = 2;
```

Comment:
FYI, this is technically a breaking change for the API. However, I believe we haven't tagged any release with this enum yet so we're OK. If we had, then we would have to use an alias (see [here](https://protobuf.dev/programming-guides/proto3/#enum)).

Reply:
yeah, we don't have any tagged release yet.